로그인&회원가입.js

```diff
@@ -0,0 +1,105 @@
+document.addEventListener("DOMContentLoaded", function () {
+  const emailInput = document.getElementById("email");
+  const passwordInput = document.getElementById("password");
+  const verifypasswordInput = document.getElementById("verifypassword");
+  const loginButton = document.getElementById("login_box");
+
+  const emailError = document.getElementById("error_email");
```

Comment:
P2:
해당 변수는 errorMessage가 들어갈 요소이니 해당 변수의 타입이 잘보이도록 아래와 같은 이름이면 더 좋을 것 같습니다.

```suggestion
  const emailErrorContainer = document.getElementById("error_email");
  const emailErrorElement = document.getElementById("error_email");
```